primitives/consensus/babe/src/digests.rs

```diff
@@ -41,7 +41,7 @@ use sp_std::vec::Vec;
 /// (VRF based) and to a secondary (slot number based).
 #[cfg(feature = "std")]
 #[derive(Clone, Debug)]
-pub enum BabePreDigest {
+pub enum PreDigest {
```

Comment:
There are some naming changes. This is opinioned, but the rationale is that:

* `digest` mod is renamed to `digests`. This is to keep it consistent with `inherents` mod.
* `BabePreDigest` is renamed to `PreDigest`. We usually only use one engine at a time, and thus it is not common to have naming collisions of multiple `PreDigest`. Plus, it keeps it consistent with `NextEpochDescriptor`.